symphony/server/typescript/describe.ts

```diff
@@ -119,7 +130,10 @@ function generateSchema(sourceFile: ts.SourceFile, fileName: string) {
       }
     }
 
-    if (ts.isFunctionDeclaration(node)) {
+    if (
+      ts.isFunctionDeclaration(node) ||
+      (ts.isVariableStatement(node) && hasConstArrowFunction(node))
```

Comment:
`isVariableStatement` check not strictly needed here but prevents visiting every node child in the AST